src/services/branch-manager.js

```diff
@@ -42,6 +43,16 @@ function createBranch(branchName, environmentSecret) {
     .send({ branchName: encodeURIComponent(branchName) });
 }
 
+function switchBranch({ id }, environmentSecret) {
+  const authToken = authenticator.getAuthToken();
+
+  return agent
+    .put(`${serverHost()}/api/environments`)
+    .set('Authorization', `Bearer ${authToken}`)
+    .set('forest-secret-key', `${environmentSecret}`)
+    .send(EnvironmentSerializer.serialize({ currentBranchId: id }));
```

Comment:
It's pretty strange to call the serializer here to build an environment just with a `currentBranchId` no ?

Reply:
It has been added to be consistent with other API call: AFAIK, each API call have to use a serializer, example here:

```
  this.updateEnvironment = async () => {
    const authToken = authenticator.getAuthToken();
    return agent
      .put(`${serverHost()}/api/environments/${config.environmentId}`)
      .set('Authorization', `Bearer ${authToken}`)
      .send(EnvironmentSerializer.serialize({
        name: config.name,
        apiEndpoint: config.url,
      }));
  };
```

Reply:
Side note: `PUT` is currently used, a REST keeper would consider it should have been a `PATCH` since it already accepts partial update, but I did not want to refactor this whole part.

Reply:
As we discuss it's a little bit overkill but keep it like that.